packages/eventbridge/src/errors.ts

```diff
@@ -0,0 +1,23 @@
+export class InternalServerError extends Error {
+  readonly details: any
+
+  constructor(details?: any) {
+    super("Internal Server Error")
+
+    this.details = details
+  }
+
+  toAppSyncError() {
+    return {
+      type: "InternalServerError",
+      message: this.message,
+    }
+  }
+
+  toHttpError() {
+    return {
+      statusCode: 500,
+      body: this.message,
+    }
+  }
```

Comment:
I think this made it over from the internal codebase by mistake.

Reply:
you're right. removed.